pennylane/io/qualtran_io.py

```diff
@@ -55,19 +58,384 @@
 
 @singledispatch
 def _get_op_call_graph(op):
+    """Return call graph for PennyLane Operator. If the call graph is not implemented,
+    return ``None``, which means we will build the call graph via decomposition"""
+
     # TODO: Integrate with resource operators and the new decomposition pipelines
-    """Return call graphs for given PennyLane Operator"""
     return None
```

Comment:
Should this raise an error if the type of `op` is not registered to any call_graph?

Reply:
No, there's no error. If the op is not registered to a call_graph we will decompose it. If there is no decomposition, then we raise an error.